public/lab-files/Lesson02/Challenges/blog-post.html

```diff
@@ -0,0 +1,30 @@
+<span>Ut, inventore, dolorum tenetur iure pariatur consequatur aliquid aut omnis nisi repudiandae repellat hic minus quam assumenda deserunt voluptas eos quasi necessitatibus recusandae totam ducimus incidunt eveniet dolor cumque id.</span>
+<span>Nobis, quos repellendus optio enim architecto quibusdam quam! Sint, praesentium, debitis, dolorem vitae tempora reiciendis placeat odit et sed officiis illum molestias corrupti labore eaque alias rem suscipit commodi iusto.</span>
+<span>Unde, recusandae, illum, velit non autem laudantium blanditiis expedita excepturi ducimus cumque necessitatibus veniam consequatur accusamus fugiat dolorum minima provident ad nemo atque voluptates est ipsam sequi deleniti quibusdam voluptatum.</span>
+<span>Fugit, vero, harum amet magni deserunt explicabo sint aliquid eligendi neque itaque quis hic reprehenderit provident culpa atque aut alias sed necessitatibus totam quasi saepe minus sit consequatur excepturi distinctio.</span>
+<span>Ipsa, aut, impedit autem eum veniam cum quis pariatur iste voluptatibus consectetur alias repellendus odit deleniti earum a quo voluptatum tempore ut saepe voluptates! Cumque, suscipit velit numquam similique harum.</span>
+<span>Nihil, voluptates perferendis asperiores consequatur autem ipsa sequi ut veritatis laudantium. Autem, assumenda, alias eveniet ipsa explicabo sint aliquid voluptatum animi ex veniam rem tempora quos facere eum error debitis.</span>
+<span>Sapiente, impedit, voluptates? Ut, voluptate blanditiis voluptates provident labore in voluptas et! Officiis, dolores, est animi molestias neque ducimus cum minus laudantium odit debitis! Sequi repellat voluptas rerum cum iusto.</span>
+<span>Obcaecati, excepturi, itaque nam ea hic rem tenetur. Expedita, debitis, adipisci, rerum tempora perspiciatis neque molestiae illo modi possimus cupiditate nam quisquam autem quibusdam minima aperiam architecto molestias nemo perferendis?</span>
+<span>Voluptas, perspiciatis et earum? At, sapiente, in, nesciunt porro optio laudantium atque ullam fuga libero molestiae magni aliquam recusandae pariatur nam eveniet tenetur et nemo neque? Nemo recusandae aperiam accusantium.</span>
+<span>Temporibus, similique, consequuntur ipsam harum sit illum minima animi impedit id quas! Aspernatur, iste, cupiditate, eum fugit quas qui fuga provident repellendus tempore nostrum obcaecati inventore quasi minima incidunt id.</span>
+<span>Mollitia, cupiditate eveniet culpa ipsum maiores iste veniam enim id nam iure ea libero porro quisquam error minima tenetur nisi quos dignissimos. Tempora, quis, debitis at maxime nobis optio labore?</span>
+<span>Ducimus nobis officia est impedit maiores ut fugiat culpa rerum magnam nisi! Obcaecati, dolorem, vel eius voluptatem consectetur earum temporibus ipsa molestiae necessitatibus omnis a aut impedit beatae enim ad?</span></p></body>
+</html>
```

Comment:
Although there is no error, I would avoid using too many span tags unless they serve a purpose. Visually, they do nothing to the code if just left alone. 

Great job overall!